src/solver/conv_asm_1x1u.cpp

```diff
@@ -308,6 +308,9 @@ bool PerformanceConfigConvAsm1x1U::IsValid(const ConvolutionContext& config) con
 
 void PerformanceConfigConvAsm1x1U::HeuristicInit(const ConvolutionContext& config)
 {
+    if(config.in_data_type == miopenDouble)
```

Comment:
It seems like this change is not needed. `IsApplicable()` of this Solver should refuse configs with double.

Reply:
?